README.md

```diff
@@ -4,4 +4,9 @@ Module 1:creating Hello Word Exercise
 .A .css file that includes one class with styles for your header (make it in your favorite color).
 .A test.md file with the text "This file should be ignored by git".
 .Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-.Open a pull request
\ No newline at end of file
+.Open a pull request
```

Comment:
- [ ] Please you should set up the Linters as it helps in enforcing better code standards. You can follow this [link](https://github.com/microverseinc/linters-config) to set up linters for the HTML & CSS projects.

Reply:
I have set up the Linters

Reply:
@vabuyia okay, when you are done with the other requested changes, request for a review on your dashboard.

Remember, yours Linter checks must ALL pass. All should be green, no ❌.